habitat-lab/habitat/config/benchmark/rearrange/play_spot.yaml

```diff
@@ -0,0 +1,42 @@
+# @package _global_
```

Comment:
Is benchmark the correct location for this?
@vincentpierre what is a recommended location for such a config? Maybe the examples/ folder in Habitat-lab?